### PR TITLE
Update Indonesian translation for version 2.0

### PR DESCRIPTION
**Pull request by harkce**
Implement https://github.com/suve/vrms-rpm/issues/38

### CODE REVIEW

File: lang/id.po [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+msgstr "Penggunaan: vrms-rpm [opsi]\n"
+
+msgid "HELP_OPTION_ASCII\n"
+msgstr "    Tampilkan rms ASCII-art ketika tidak ditemukan paket non-free,\n"
+       "    atau ketika terdapat paket non-free lebih dari 10%% dari total.\n"
+
+msgid "HELP_OPTION_COLOUR\n"
+msgstr "    Mengontrol apakah terminal escape sequences dipergunakan\n"
+       "    untuk mewarnai output. Nilai defaultnya 'auto', yang mana menggunakan output berwarna\n"
+       "    ketika menulis di terminal, tetapi tidak berlaku apabila menulis di file atau pipe.\n"
```
Comment (suve) [inline lang/id.po]: Each line of the translated text should fit within 80 characters. You don't have to fit the text in 3 lines, though - you can split it into 4 lines or more, if needed.

File: lang/id.po [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+msgid "HELP_OPTION_EXPLAIN\n"
+msgstr "    Ketika mendaftar paket, tampilkan lisensinya\n"
+       "    untuk memastikan apakah termasuk klasifikasi free / non-free.\n"
+
+msgid "HELP_OPTION_HELP\n"
+msgstr "    Menampilkan bantuan dan keluar.\n"
+
+msgid "HELP_OPTION_IMAGE\n"
+msgstr "    Seperti --ascii, tetapi menampilkan gambar menggukanak blok karakter Unicode\n"
+       "    dan kode terminal escape mode 256-warna.\n"
```
Comment (suve) [inline lang/id.po]: Make the first line fit in the 80-character limit.

File: lang/id.po [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+# A place for you to credit yourself!
+# Put something like "Latin translation by Jvlivs Gaivs\n" here.
+msgid "TRANSLATION_AUTHOR\n"
+msgstr "Diterjemahkan ke Bahasa oleh Habib Fikri\n"
+
+msgid "FREE_PACKAGES_COUNT\n"
+msgid_plural "FREE_PACKAGES_COUNT\n"
+msgstr[0] "%d paket free (dari total %s)\n"
+msgstr[1] "%d paket free (dari total %s)\n"
```
Comment (suve) [inline lang/id.po]: Just to make sure - are these the proper plural forms for Indonesian? `msgstr[0]` refers to a single free package, `msgstr[1]` to multiple.

File: man/id.man [hunk 4]
```diff
@@ -0,0 +1,93 @@
+.TH VRMS-RPM 1 "2018-05-18"
```
Comment (suve) [inline man/id.man]: Change this to match the date of the translation.

File: lang/id.po (hunk 3 above)
Comment (harkce) [reply lang/id.po]: Yes, it is. In Bahasa, if a noun prefixes with its amount, the form is the same as its singular form.

File: lang/id.po (hunk 1 above)
Comment (suve) [reply lang/id.po]: Sorry, I didn't make it clear enough - this is 80 characters from the opening `"` to the closing `"`, i.e. including the trailing 4-space indent.

File: lang/id.po (hunk 2 above)
Comment (suve) [reply lang/id.po]: Same here - 80 characters from the opening " to the closing ", i.e. including the trailing 4-space indent.

File: lang/id.po (hunk 3 above)
Comment (suve) [reply lang/id.po]: In that case, we can remove the plural forms, it seems. Please edit the file header like this:
```
Plural-Forms: nplurals=1; plural=0;
```
And then remove `msgstr[1]` from both plural strings, leaving only `msgstr[0]`.

File: lang/id.po (hunk 1 above)
Comment (harkce) [reply lang/id.po]: Does this only apply to `HELP_OPTION`?

Comment (suve) [reply lang/id.po]: Yes. The `ERR_` messages should **not** be wrapped.

**Comment by harkce** [comment]
@suve are there still any issues?

**Comment by suve** [comment]
Looks fine now. Merged. Thanks!

**Pull request merged**